package/toltec-bootstrap/toltecctl

```diff
@@ -460,7 +463,7 @@ reinstall-root() {
         if [[ -v "on_root_packages[$pkgname]" ]]; then
             reinstall_packages[$pkgname]=1
         fi
-    done < <(gunzip -c /opt/var/opkg-lists/* | grep "^Package:" | awk '{print $2}')
+    done < <(gunzip -c $toltec_dest/var/opkg-lists/* | grep "^Package:" | awk '{print $2}')
```

Comment:
I don't think anyone would install toltec in a path with whitechars, but it's always wise to put paths from env variables inside ""

Reply:
I'm surprised our linter missed this, it catches all the other ones.